tests/models/test_bart.py

```diff
@@ -168,3 +168,68 @@ def test_models(
             name_1="vllm",
             num_outputs_0_skip_tokens=hf_skip_tokens,
         )
+
+    @pytest.mark.parametrize("model", ["facebook/bart-large-cnn"])
+    @pytest.mark.parametrize("dtype", ["bfloat16"])
+    @pytest.mark.parametrize("max_tokens", [2048])
+    @pytest.mark.parametrize("num_logprobs", [5])
+    @pytest.mark.parametrize("decoder_prompt_type", list(DecoderPromptType))
+    def test_model_with_cuda_graph(
```

Comment:
I think having an end-to-end encoder/decoder+CUDAGraph test with a model is great.

However, I observe that CUDAGraph support does not seem to require model changes. Therefore, if one model works with CUDAGraph, then all models work. So for example, hypothetically, you would not need to have a "T5 with CUDAGraph" test or a "Whisper with CUDAGraph" test, as these would be redundant, right?

So this is not really a BART-specific test, but rather an end-to-end test of the vLLM pipeline with CUDAGraph, which happens to employ BART as the model under test.

Does it make sense to move this test into a location that is not specific to a single model? Perhaps it could be moved to

* `tests/models/test_models.py`
* its own file (i.e. `tests/models/test_models.py`)
* its own folder (i.e. `tests/cudagraph/test_cudagraph.py` and add a `test_e2e_encoder_decoder_cudagraph`)